File: Maths/snack.py
# AtCoder Beginner Contest (Problem 148)

import math

class Solution(object):
    def lcm(self,a:int,b:int)->int:
        return (a*b)//math.gcd(a,b)



if __name__=='__main__':
    a = 123
    b = 456
    s = Solution()
    print(s.lcm(a,b)) # 18696

File: Maths/snack.cxx
// AtCoder Beginner Contest 148 (Problem C)

#include <bits/stdc++.h>
using namespace std;

class Solution{
    public:
    static int lcm(int a, int b){
        return (a*b)/__gcd(a,b);
    }
};
int main(){
    ios_base::sync_with_stdio(0);
    int a = 123;
    int b = 456;
    cout << Solution::lcm(a,b); // 18696
}
